connexion/apis/aiohttp_api.py

```diff
@@ -269,13 +271,44 @@ def get_request(cls, req):
         :rtype: ConnexionRequest
         """
         url = str(req.url)
-        logger.debug('Getting data and status code',
-                     extra={'has_body': req.has_body, 'url': url})
+        content_type = req.content_type
+
+        logger.debug(
+            'Getting data and status code',
+            extra={
+                # has_body | can_raed_body report if
+                # body has been read or not
+                # body_exists refers to underlying stream of data
+                'has_body': req.body_exists,
+                'can_ready_body': req.can_read_body,
+                'content_type': content_type,
+                'url': url,
+            },
+        )
 
         query = parse_qs(req.rel_url.query_string)
         headers = req.headers
         body = None
-        if req.body_exists:
+
+        # if request is not multipart, `data` will be empty dict
+        post_data = yield from req.post()
+        files = {}
+        form = {}
+        if post_data:
+            logger.debug('Reading multipart data from request')
+            for k, v in post_data.items():
+                if isinstance(v, web.FileField) and k not in files:
+                    files[k] = v
+                elif isinstance(v, web.FileField) and k in files:
+                    try:
```

Comment:
Any concerns of using `collections.defaultdict` instead of handling an exception?

Reply:
TBH I have doubts if that is correct. It works for many files but if I mix normal fields and files everything goes wrong.

Also form is processed in weird way, I will post details łatwe for that.

Reply:
Interesting thing. I noticed that if you try to submit multiple fields for Flask it will ignore them and take just the first value.
I might want to create an issue on its own for that :)

Reply:
FYI: https://github.com/zalando/connexion/issues/992